satpy/readers/fci_l1c_nc.py

```diff
@@ -84,15 +84,15 @@
 .. _test data release: https://www.eumetsat.int/simulated-mtg-fci-l1c-enhanced-non-nominal-datasets
 """
 
-from __future__ import (division, absolute_import, print_function,
-                        unicode_literals)
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Not sure why this line was changed, but these future directives are all redundant now.

Reply:
PR welcome